static/js/panels.js

```diff
@@ -78,8 +79,23 @@ export function show_profile_incomplete(is_profile_incomplete) {
     }
 }
 
+export function is_timezone_inconsistent() {
+    const browser_timezone = new Intl.DateTimeFormat().resolvedOptions().timeZone;
```

Comment:
I think `new Intl.DateTimeFormat().resolvedOptions().timeZone;` could gointo a separate function since you are using it in two places.